patroni/postgresql.py

```diff
@@ -668,7 +668,7 @@ def restart(self):
             self.call_nowait(ACTION_ON_RESTART)
         else:
             self.set_state('restart failed ({0})'.format(self.state))
-        return ret
+        return True
```

Comment:
I think that would break the code in ha.py(namely in def restart) that checks whether the restart returns False in order to report the issue to the client.